design-library/source/_patterns/02-molecules/05-footer-list/footer-image-list.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please either rename the file/pattern to `footer-image-list` or make a new pattern for `footer-image-list`